src/security/intel/txt/common.c

```diff
@@ -601,7 +601,7 @@ bool intel_txt_prepare_txt_env(void)
 			break;
 		}
 	}
-
+#if CONFIG(TPM2)
```

Comment:
Regular `if (CONFIG(TPM2))` doesn't work?

Reply:
@miczyg1 nope, I get an `implicit declaration of function` error, since `check_tpm2_indices()` is declared within `#IF` [here](https://github.com/Dasharo/coreboot/pull/541/files/42719d557ef0ac4a9b064d7e331a6561676850a2#diff-b06cd9106ae1c9f5ccc2b489c91a3c6e4fe058b94313f4b3ac851c59262ac36fR417)